### PR TITLE
Rename testgroups, fix travis.

### PR DESCRIPTION
**Pull request by spxtr**
I wish the jenkins_verify test ran under bazel...

### CODE REVIEW

File: testgrid/config/config.yaml [hunk 1]
```diff
@@ -860,11 +860,11 @@ test_groups:
 # bazel, run on prow
 - name: ci-kubernetes-bazel-build
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-bazel-build
-- name: ci-kubernetes-bazel-build-1.6
+- name: ci-kubernetes-bazel-build-1-6
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-bazel-build-1.6
```
Comment (ixdy) [inline testgrid/config/config.yaml]: this also needs to be fixed?

**Comment by krzyzacy** [comment]
ehhhahhh I was mad cannot find what was wrong...

yeah the plan is moving it to bazel once we get rid of jenkins-job-builder :-(

File: testgrid/config/config.yaml (hunk 1 above)
Comment (spxtr) [inline testgrid/config/config.yaml]: Done.

**Comment by ixdy** [comment]
/lgtm

is the "no-dots-in-job-names" a prow restriction?

**Comment by ixdy** [comment]
@krzyzacy how is bazel support a prerequisite to autogenerating the testgrid config?

**Comment by spxtr** [comment]
> is the "no-dots-in-job-names" a prow restriction?

Yes, see #2377. K8s pods can't have dots in their names.

> how is bazel support a prerequisite to autogenerating the testgrid config?

He means that the travis test will live on travis until we're off of Jenkins.

**Comment by krzyzacy** [comment]
@ixdy that yet to know exactly how but we'll have a solution when we get there.

**Pull request merged**